iothub/service/src/ExportImportDevice.cs

```diff
@@ -11,27 +11,51 @@
 namespace Microsoft.Azure.Devices
 {
     /// <summary>
-    ///  contains device properties specified during export/import operation
+    /// Contains device properties specified during export/import operation
     /// </summary>
     public sealed class ExportImportDevice
     {
         private string _eTag;
         private string _twinETag;
 
         /// <summary>
-        /// making default ctor public
+        /// Property container
+        /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1034:Nested types should not be visible", Justification = "Public property. No behavior changes allowed.")]
+        public sealed class PropertyContainer
+        {
+            /// <summary>
+            /// Desired properties
+            /// </summary>
+            [JsonProperty(PropertyName = "desired", NullValueHandling = NullValueHandling.Ignore)]
+            public TwinCollection DesiredProperties { get; set; }
+
+            /// <summary>
+            /// Reported properties
+            /// </summary>
+            [JsonProperty(PropertyName = "reported", NullValueHandling = NullValueHandling.Ignore)]
+            public TwinCollection ReportedProperties { get; set; }
+        }
+
+        /// <summary>
+        /// Create an ExportImportDevice <see cref="ExportImportDevice" />
         /// </summary>
         public ExportImportDevice()
         {
         }
 
         /// <summary>
-        /// ctor which takes a Device object along with import mode
+        /// Create an ExportImportDevice <see cref="ExportImportDevice" />
```

Comment:
@azabbasi Wouldn't this show the 'ExportImportDevice' twice? I think it should be `Create an <see cref="ExportImportDevice" />`

Reply:
Yes, it will.